app/Appointments.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Appointments extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var string[]
+     */
+    protected $fillable = [
+        'client_id',
+        'date_time', 
+        'type', 
+        'price',
+        'duration',
+    ];
+
+    public function clients()
```

Comment:
Can we look at starting to add return type comments and type hinting?
```suggestion
    /**
     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
     */
    public function client(): BelongsTo
    {
        return $this->belongsTo(Clients::class);
    }
```
Also, this method should be singular, as there is only one client.